src/Components/Gallery/Gallery.js

```diff
@@ -23,8 +23,6 @@ import { Api, ArtSvc } from '../../api';
 import ImageModal from '../ImageModal/ImageModal';
 
 const Gallery = () => {
-  const [isLoading, setIsLoading]               = useState(true);
-  const [isLoadingMore, setIsLoadingMore]       = useState(true);
   const [searchParams, setSearchParams]         = useState({});
```

Comment:
`isLoading` and `isLoadingMore` were not being used?

Reply:
Nope, the `<InfiniteScroll/>` component takes care of the loader now